tests/ftpserver.json

```diff
@@ -0,0 +1,6 @@
+{
+    "ftpserver": "nexycrypt.96.lt",
```

Comment:
What is that? A service to create temporary servers?

Reply:
It looks like a web server just created by you. Am I right?

Reply:
Yes, I just want to do the real testing so I create this FTP server.
But I'm not sure this FTP server is available now.

Reply:
It is. And I'm sure you understand we can't rely on a real external server.

Plus, you should change the credentials, this is not secure at all. :confused: 

IMO we should just mock the http request. Also see #7

Reply:
According to this [post](https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables), perhaps we can let the FTP user name and password being the encrypted environment variables.

What do you think?Or just use the mock to pass this testing?

Reply:
I think we should use mock for global testing. For that, you should wait #7 to be merged and use this: https://github.com/php-http/mock-client

At least, we may create a single real test with HTTP challenge using [ngrok](https://ngrok.com/).

I'll also try to setup simple test on my side for the bridge.